eventbus/src/test/java/pl/edu/uj/kimage/eventbus/EventA.java

```diff
@@ -0,0 +1,4 @@
+package pl.edu.uj.kimage.eventbus;
+
+public class EventA implements Event {
+}
```

Comment:
Hold on... This is a partial commit, and this class doesn't exist any more. Shall I squash commits?